projects/plugins/jetpack/modules/related-posts/jetpack-related-posts.php

```diff
@@ -1733,43 +1820,48 @@ protected function _enqueue_assets( $script, $style ) {
 			);
 			wp_localize_script( 'jetpack_related-posts', 'related_posts_js_options', $related_posts_js_options );
 		}
-		if ( $style ){
+		if ( $style ) {
 			wp_enqueue_style( 'jetpack_related-posts', plugins_url( 'related-posts.css', __FILE__ ), array(), self::VERSION );
 			wp_style_add_data( 'jetpack_related-posts', 'rtl', 'replace' );
 			add_action( 'amp_post_template_css', array( $this, 'render_amp_reader_mode_css' ) );
 		}
 	}
 
+	/**
+	 * Render AMP's reader mode CSS.
+	 */
 	public function render_amp_reader_mode_css() {
-		echo file_get_contents( plugin_dir_path( __FILE__ ) . 'related-posts.css' );
+		echo file_get_contents( plugin_dir_path( __FILE__ ) . 'related-posts.css' ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped -- this is loading a CSS file.
```

Comment:
The sniff complaining about `wp_remote_get()` only applies when the thing being fetched might be a URL. When we're fetching a filesystem path, it's fine to use `file_get_contents()`. Unfortunately the sniff sometimes can't tell the difference.

I'd hoped it could tell the difference with `__DIR__`, but apparently it only looks for WordPress's various constants. Sigh.

Reply:
Yeah I had a feeling something was up, I didn't do enough digging on the differences. Changing it back to file_get_contents and ignoring the sniff.